server/auth/types/jwt/jwt_auth.ts

```diff
@@ -63,7 +61,7 @@ export class JwtAuthentication extends AuthenticationType {
     return undefined;
   }
 
-  private getBearerToken(request: OpenSearchDashboardsRequest): string | undefined {
+  public getBearerToken(request: OpenSearchDashboardsRequest): string | undefined {
```

Comment:
why changing all these functions to public?

Reply:
Addressed.